pom.xml

```diff
@@ -1797,11 +1808,8 @@
 
         <carbon.registry.imp.pkg.version>0.0.0</carbon.registry.imp.pkg.version> <!-- todo remove this and use {carbon.kernel.imp.pkg.version} after registry bundles are corrected to export correct versions-->
 
-        <mongo-java-driver.orbit.version>3.5.0</mongo-java-driver.orbit.version>
-        <mongo-java-driver.orbit.imp.pkg.version>[3.4.1,4.0.0)</mongo-java-driver.orbit.imp.pkg.version> <!--todo change this to appropriate version range ([2.11.4.wso2v1,2.12.0)) after osgi bundle is corrected - rajith-->
-
-        <jongo.orbit.version>1.4.0.wso2v1</jongo.orbit.version>
-        <jongo.orbit.imp.pkg.version>[1.4,1.5)</jongo.orbit.imp.pkg.version>
+        <mongodb-driver-sync.orbit.version>4.9.1</mongodb-driver-sync.orbit.version>
```

Comment:
Are we using an orbit for the new version? If not let's rename this without `orbit`

Reply:
Fixed with [2a5b752](https://github.com/wso2/product-micro-integrator/pull/3859/commits/2a5b7522ce2cd6886646e94edf5c01ebb4c19d01)